app/controllers/group_memberships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GroupMembershipsController < AuthenticationController
+  def create
+    result = CreateGroupMembership.call(user_id:  current_user.id,
+                                        group_id: params[:group_id])
```

Comment:
I can think the endpoint could be something like:

```
POST /groups/:group_id/memberships
```

That will include the group id within the endpoint itself.